typescript/infra/src/tx/govern-transaction-reader.ts

```diff
@@ -448,11 +456,14 @@ export class GovernTransactionReader {
     chain: ChainName,
     tx: AnnotatedEV5Transaction,
   ): boolean {
-    return (
-      tx.to !== undefined &&
-      timelocks[chain] !== undefined &&
-      eqAddress(tx.to!, timelocks[chain]!)
-    );
+    const isNewTimelock =
+      this.timelocks[chain] !== undefined &&
+      eqAddress(tx.to!, this.timelocks[chain]!);
+    const isLegacyTimelock =
+      legacyTimelocks[chain] !== undefined &&
+      eqAddress(tx.to!, legacyTimelocks[chain]!);
+
+    return tx.to !== undefined && (isNewTimelock || isLegacyTimelock);
```

Comment:
alternative option could've been to do it like "isProxyAdminFromBytecode" where we compare the calldata with expected function selectors https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/1f3f2f807ff2ce7dfe99d8048a21cf01a43bd655/typescript/sdk/src/deploy/proxy.ts#L155